proto/zipkin.proto

```diff
@@ -0,0 +1,213 @@
+// Copyright 2018 The OpenZipkin Authors
```

Comment:
Can this be sourced from `zipkin-api`? Even if the format doesn't change often, it's easier to manage as a dependency.

Reply:
Great suggestion, thanks!
I've asked the Haystack folks to do the same: https://github.com/ExpediaDotCom/haystack-idl/issues/34